sdk/keyvault/azure-keyvault-certificates/tests/test_examples_certificates_async.py

```diff
@@ -50,8 +50,18 @@ class TestExamplesKeyVault(AsyncKeyVaultTestCase):
         create_certificate_poller = await certificate_client.create_certificate(name=cert_name, policy=cert_policy)
 
         # awaiting the certificate poller gives us the result of the long running operation
-        create_certificate_result = await create_certificate_poller
-        print(create_certificate_result)
+        certificate = await create_certificate_poller
+        print(certificate.id)
+        print(certificate.name)
+        print(certificate.policy.key_properties.exportable)
+        print(certificate.policy.key_properties.key_type)
+        print(certificate.policy.key_properties.key_size)
+        print(certificate.policy.key_properties.reuse_key)
+        print(certificate.policy.content_type)
+        print(certificate.policy.issuer_name)
+        print(certificate.policy.subject_name)
+        print(certificate.policy.san_dns_names)
+        print(certificate.policy.validity_in_months)
```

Comment:
Lots of printing here, at a glance I naively think `certificate.id`, `.name` and `.issuer_name` are probably the most universally interesting?

Reply:
sounds good, I am just printing .id, .name, and .properties.issuer_name now. Thanks!